app/controllers/admin/oral_history_access_requests_controller.rb

```diff
@@ -9,7 +9,8 @@ def index
     unless status == "any" || status.blank?
       scope = scope.where(delivery_status: status)
     end
-    @oral_history_access_requests = scope.where('created_at > ?', 3.months.ago).order(created_at: :desc).to_a
+    @oral_history_access_requests = scope.order(created_at: :desc).page(params[:page]).per(100)
```

Comment:
We were showing way more than 100 before I think, should we make it 200 or 300 any thoughts?